Makefile.am

```diff
@@ -17,8 +17,8 @@ doxygen_doc_files = $(noinst_HEADERS)
 perftest_dir = $(pkgdatadir)/perftest
 doc_dir = $(pkgdatadir)/doc
 
-dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
-dist_doc__DATA = README
+dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README.md contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
```

Comment:
README.md does not comply to gnu coding convention.
we should have README.md for github only and use README for autotools.